pyseir/rt/infer_rt.py

```diff
@@ -91,19 +94,32 @@ def _generate_input_data(
     include_testing_correction: bool,
     include_deaths: bool,
     figure_collector: Optional[list],
-):
+) -> pd.DataFrame:
     """
     Allow the RtInferenceEngine to be agnostic to aggregation level by handling the loading first
 
     include_testing_correction: bool
         If True, include a correction for testing increases and decreases.
     """
     # TODO: Outlier Removal Before Test Correction
-    times, observed_new_cases, observed_new_deaths = load_data.calculate_new_case_data_by_region(
-        regional_input.get_timeseries(),
-        t0=InferRtConstants.REF_DATE,
-        include_testing_correction=include_testing_correction,
-    )
+    try:
+        (
+            times,
+            observed_new_cases,
+            observed_new_deaths,
+        ) = load_data.calculate_new_case_data_by_region(
+            regional_input.get_timeseries(),
+            t0=InferRtConstants.REF_DATE,
+            include_testing_correction=include_testing_correction,
+        )
+    except AssertionError as e:
```

Comment:
Oh, sorry you needed this. Let me know what fix you'd like from https://covidactnow.slack.com/archives/C0110QQAPJA/p1598547382041700?thread_ts=1598483626.040100&cid=C0110QQAPJA

Reply:
Right now I am going to filter out all fips that end in "999" upstream. So this additional context shouldn't be necessary going forward. I'm going to leave it in for right now, but plan to remove it when I fix the MA counties issues too.